### PR TITLE
#4 Containerization of the Service

### PR DESCRIPTION
**Pull request by raghul2304**
[Story link](https://tree.taiga.io/project/zubae-payment-gateway/epic/2) 

## Description
- Created the Docker file for both auth and profile service 
- Created the compose file  

## Checklist
- [x] Check the Docker file for auth and profile service 
- [x] Check the docker-compose file for this services

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
+FROM golang:latest
```
Comment (raam-techie) [inline Dockerfile:1]: Please specify the Go version used in our application

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,13 @@
+FROM golang:latest 
+
+WORKDIR /app 
+
+COPY . /app 
+
+WORKDIR /app
```
Comment (raam-techie) [inline Dockerfile:7]: Remove line no:7 WORKDIR  /app [can perform all the necessary operations within a single working directory]

File: docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+version: '3'
+
+services:
+  identity_service:
+    build: .
+    ports:
+      - "8080:8080"
```
Comment (afrze) [inline docker-compose.yml:7]: Make to add one empty line in all files. This is common for any file, in any language

**Review by afrze — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 2 above)
Comment (raghul2304) [reply Dockerfile]: changes done

File: docker-compose.yml (hunk 3 above)
Comment (raghul2304) [reply docker-compose.yml]: changes done

**Review by raam-techie — APPROVED** [review]

**Review by Jeyashree46 — APPROVED** [review]

**Review by afrze — APPROVED** [review]
Look good to me 🚀

**Pull request merged**